code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -401,9 +393,64 @@
 		if(AI_IDLE)
 			if(FindTarget(possible_targets, 1))
 				. = 1
-				AIStatus = AI_ON //Wake up for more than one Life() cycle.
+				toggle_ai(AI_ON) //Wake up for more than one Life() cycle.
 			else
 				. = 0
 
 /mob/living/simple_animal/hostile/proc/AIShouldSleep(var/list/possible_targets)
 	return !FindTarget(possible_targets, 1)
+
+/mob/living/simple_animal/proc/toggle_ai(togglestatus)

```

Comment:
also changed on TG. Should also probably be in `simple_animals.dm` because it's that level proc.